test/boost/reusable_buffer_test.cc

```diff
@@ -6,64 +6,72 @@
  * SPDX-License-Identifier: AGPL-3.0-or-later
  */
 
```

Comment:
Also a reference in this patch's commit message. Please create a new issue to fix.

Reply:
Created https://github.com/scylladb/scylladb/issues/13437

Reply:
Done.